### ISSUE TITLE
Voice message playback slider not working

### ISSUE THREAD

**Issue by SurendraG1985** [UnigramDev/Unigram#2351]
Screen reader user cannot by any means rewind/fast forward the currently playing voice message, as the slider available for the same is non-functional using keyboard. Screen reader just keeps reading the playback ProgressBar, as focus is set to this slider. Note that this issue is for music files as well.
